Formula/mitie.rb

```diff
@@ -34,8 +34,7 @@ def install
     include.install Dir["mitielib/include/*"]
     lib.install "mitielib/#{shared_library("libmitie")}", "mitielib/libmitie.a"
 
-    xy = Language::Python.major_minor_version "python3"
-    (lib/"python#{xy}/site-packages").install "mitielib/mitie.py"
+    (lib/Language::Python.site_packages("python3.10")).install "mitielib/mitie.py"
```

Comment:
```suggestion
    (prefix/Language::Python.site_packages("python3.10")).install "mitielib/mitie.py"
```
`Language::Python.site_packages("python3.10")` is `"lib/python3.10/site-packages"`.